docs/source/reference/series.rst

```diff
@@ -133,6 +133,52 @@ Reshaping, sorting, transposing
    Series.sort_index
    Series.sort_values
 
+Accessors
+---------
+
+Koalas provides dtype-specific methods under various accessors.
+These are separate namespaces within :class:`Series` that only apply
+to specific data types.
+
+========= =========================
+Data Type                  Accessor
+========= =========================
+Datetime  :ref:`dt <api.series.dt>`
+========= =========================
+
+.. _api.series.dt:
+
+Datetimelike Properties
+~~~~~~~~~~~~~~~~~~~~~~~
+
+``Series.dt`` can be used to access the values of the series as
+datetimelike and return several properties.
+These can be accessed like ``Series.dt.<property>``.
+
+Datetime Properties
+^^^^^^^^^^^^^^^^^^^
+
+.. currentmodule:: databricks.koalas.series
+.. autosummary::
+   :toctree: api/
+
+   Series.dt.date
+   Series.dt.year
+   Series.dt.month
+   Series.dt.week
+   Series.dt.weekofyear
+   Series.dt.day
+   Series.dt.dayofweek
+   Series.dt.weekday
+   Series.dt.dayofyear
+   Series.dt.hour
+   Series.dt.minute
+   Series.dt.second
+   Series.dt.millisecond
+   Series.dt.microsecond
+
```

Comment:
I know this is from the previous one, but I'm wondering why there is a blank line here.

Reply:
oh, seems like `strftime` doesn't exist in pandas' document?